Devscord.DiscordFramework/Framework/Architecture/Controllers/IgnoreForHelp.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace Devscord.DiscordFramework.Framework.Architecture.Controllers
+{
+    [AttributeUsage(AttributeTargets.Method)]
+    public class IgnoreForHelp : Attribute
```

Comment:
to powinno być konfigurowalne z poziomu konfiguracji 
a nie zapisywane na sztywno w kodzie